.moai/release/RELEASE-NOTES-v2.16.0.md

```
@@ -0,0 +1,184 @@
+moai update
+```
+
+`moai update` 실행 시:
+- 신규 6 cookbook rule 파일이 `.claude/rules/moai/development|quality|workflow/`로 자동 배포
+- 기존 user customization (`.moai/project/`, `.moai/specs/`) 보호
+- `.moai/config/sections/runtime.yaml` 신규 (Token Circuit Breaker)
+- `.moai/config/sections/harness.yaml` 신규 (V3R3 Phase A에서 도입)
+- `.claude/agent-memory/` 4-type taxonomy 점검 (warning만, 비차단)
+
+업그레이드 후 권장 검증:
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)